### PR TITLE
Adds the copy for the Linux prereqs.

### PR DESCRIPTION
**Pull request by ibm-jason**
This PR adds the Linux prereqs copy for istioctl tab completion to address the PR review by @ericvn posted [here](https://github.com/istio/istio.io/pull/4023#discussion_r285142879).

Signed-off-by: Jason Clark <jason.clark@ibm.com>

### CODE REVIEW

File: content/docs/setup/kubernetes/download/index.md [hunk 1]
```diff
@@ -103,6 +103,12 @@ Once the bash-completion package has been installed on your macOS system, also m
 `[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"`
 {{< /tip >}}
 
+
```
Comment (frankbu) [inline content/docs/setup/kubernetes/download/index.md]: circleci error (multiple blank lines)

File: content/docs/setup/kubernetes/download/index.md [hunk 2]
```diff
@@ -103,6 +103,12 @@ Once the bash-completion package has been installed on your macOS system, also m
...
+### Prerequisites for Linux
+
+For Linux, to install the bash completion package, you would use the `apt-get install bash-completion` command for Debian-based Linux distributions or `yum install bash-completion` for RPM-based Linux distributions, the two most common occurrences.
+
```
Comment (frankbu) [inline content/docs/setup/kubernetes/download/index.md]: same as above

File: content/docs/setup/kubernetes/download/index.md (hunk 1 above)
Comment (ibm-jason) [reply content/docs/setup/kubernetes/download/index.md]: fixed in latest update.

File: content/docs/setup/kubernetes/download/index.md (hunk 2 above)
Comment (ibm-jason) [reply content/docs/setup/kubernetes/download/index.md]: fixed in latest update.

**Pull request merged**